src/resqml2/AbstractGridRepresentation.h

```diff
@@ -361,6 +361,51 @@ namespace RESQML2_NS
 		*/
 		ULONG64 getCellStratigraphicUnitIndices(ULONG64 * stratiUnitIndices);
 
+		//************************************************************
+		//**************** LINK WITH ROCKFLUID ********************
+		//************************************************************
+
+		/**
+		* Set the stratigraphic organization interpretation which is associated to this grid representation.
+		* @param stratiUnitIndices	Index of the stratigraphic unit of a given stratigraphic column for each cell. Array length is the number of cells in the grid or the blocked well.
+		* @param nullValue			The value which is used to tell the association between a cell and strati unit is unavailable.
+		* @param stratiOrgInterp	The stratigraphic organization interpretation which is associated to this grid representation.
+		*/
+		void setCellAssociationWithFluidOrganizationInterpretation(ULONG64 * fluidUnitIndices, const ULONG64 & nullValue, RESQML2_0_1_NS::RockFluidOrganizationInterpretation* stratiOrgInterp);
+
+		/**
+		* @return	nullptr if no stratigraphic organization interpretation is associated to this grid representation. Otherwise return the associated stratigraphic organization interpretation.
+		*/
+		RESQML2_0_1_NS::RockFluidOrganizationInterpretation* getFluidOrganizationInterpretation() const;
```

Comment:
I think I would have preferred to keep "Rock" before "fluid". Same everywhere.
If you want to shorten the method: organization could be "org", interpretation could be "interp"